pixl_ehr/tests/test_processing.py

```diff
@@ -186,11 +257,81 @@ async def test_message_processing() -> None:
 
         assert value == expected_value
 
-    anon_row = pixl_db.execute_query_string(
-        "select * from emap_data.ehr_anon where gcs = %s", [gcs]
+    anon_all_rows = pixl_db.execute_query_string(
+        "select mrn, accession_number, xray_report from emap_data.ehr_anon where gcs = %s", [gcs]
     )
-    anon_mrn, anon_accession_number = anon_row[:2]
+    assert len(anon_all_rows) == 1
+    anon_row = anon_all_rows[0]
+    anon_mrn, anon_accession_number, anon_report_text = anon_row
     assert anon_mrn != mrn
-    assert anon_accession_number != accession_number
+    assert anon_accession_number != message.accession_number
+
+    # No deidentification performed so we expect the report to stay identical
+    assert report_text == anon_report_text
+
+
+@pytest.mark.processing()
+@pytest.mark.asyncio()
+async def test_radiology_export(example_messages) -> None:
+    """
+    GIVEN a message processed by the EHR API
+    WHEN export_radiology_as_parquet is called
+    THEN the radiology reports are exported to a parquet file and symlinked to the latest export
+    directory
+    """
+    # ARRANGE
+    message = example_messages[0]
+    project_name = message.project_name
+    extract_date = message.omop_es_timestamp
+    pe = ParquetExport(project_name, extract_date)
+    await process_message(message)
+
+    # ACT
+    export_radiology_as_parquet(project_name, omop_es_timestamp_1)
+
+    # ASSERT
+    parquet_file = pe.radiology_output / "radiology.parquet"
+
+    assert parquet_file.exists()
+    assert parquet_file.is_file()
+    # Check symlink
+    latest_parquet_file = pe.latest_parent_dir / "radiology.parquet"
+    latest_parquet_file.is_symlink()
+
+
+@pytest.mark.processing()
+@pytest.mark.asyncio()
+async def test_radiology_export_multiple_projects(example_messages) -> None:
+    """
+    GIVEN 4 messages, each from a different project+extract processed by the EHR API
+    WHEN export_radiology_as_parquet is called for 1 given project+extract
+    THEN only the radiology reports for that project are exported
```

Comment:
nice! Might be worth saying that there are two exports for the project at different datetimes so we know its checking that as well